src-tauri/src/lib.rs

```diff
@@ -510,3 +510,75 @@ pub fn get_installed_mods(game_install: GameInstall) -> Result<Vec<NorthstarMod>
 
     Ok(installed_mods)
 }
+
+async fn get_ns_mod_download_url(thunderstore_mod_string: String) -> Result<String, String> {
+
+    // TODO: This will crash the thread if not internet connection exist. `match` should be used instead
```

Comment:
So this is not ready for review yet 😄

Reply:
Eh, FlightCore will already do that if you launch it without internet :>

Reply:
I'll open a dedicated issue for this